### PR TITLE
[ACTION] Lusha #864

### PR DESCRIPTION
**Pull request by lcaresia**
@dannyroosevelt 

I create the actions based on API, but to test I will need a premium account.
Can I move this for review or I set as blocked for now?

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

**Comment by dannyroosevelt** [comment]
> @dannyroosevelt
> 
> I create the actions based on API, but to test I will need a premium account. Can I move this for review or I set as blocked for now?

please move to review, and we can figure that out during QA

File: components/lusha/actions/find-company/find-contact.mjs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import lusha from "../../lusha.app.mjs";
```
Comment (jcortes) [inline components/lusha/actions/find-company/find-contact.mjs:1]: Please rename this file to `find-company.mjs`

**Review by jcortes — CHANGES_REQUESTED** [review]
Hi @lcaresia just three minor observations everything else looks good to me. thanks

**Comment by vunguyenhung** [comment]
Hello @dannyroosevelt, I faced the same issue as @lcaresia that I can not get the API Key since Lusha require Enterprise plan for it
screenshot - 2022-06-20T215904 937

Similar to Zoom Phone and others that requiring paid plan, I suggest that we ship this first and monitor if any customer faced any issue. Then if there's a bug then we can have a paid account to investigate. what do you think?

**Comment by vunguyenhung** [comment]
Hello @lcaresia, as discussed with @dannyroosevelt, we agree to ship this Lusha component first for customer to use, then monitor if any customer faced any issue. Then if there's a bug then we can have a paid account to investigate.

cc @jcortes

**Pull request merged**